app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
```

Comment:
It is also a best practice to sort imports in alphabetical order. Check below as well